src/app/shared/keycloak.service.ts

```diff
@@ -31,6 +31,9 @@ export class KeycloakService {
             this.auth.authz = keycloakAuth;
             this.loginSubject.next(keycloakAuth.token);
             this.auth.logoutUrl = `${keycloakAuth.authServerUrl}/realms/${config.realm}/protocol/openid-connect/logout?redirect_uri=${document.baseURI}`;
+            if (window['analytics']) {
+              window['analytics'].identify(this.auth.authz.tokenParsed);
```

Comment:
I wonder if we should perhaps make a tiny change here. What if we pre-select the `email` field as the "unique id"?

Making the call:

    window['analytics'].identify(this.auth.authz.tokenParse.email, this.auth.authz.tokenParsed);

That way Segment already automatically assumes all those pages and tracks belong to the same user.